src/App.js

```diff
@@ -1,71 +1,27 @@
-import logo from './logo.svg';
-import './App.css';
```

Comment:
please add back this CSS file so the app imports and uses the Livvic font by default